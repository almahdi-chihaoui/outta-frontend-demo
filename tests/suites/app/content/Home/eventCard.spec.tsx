import React from "react";
import renderer from "react-test-renderer";

import EventCard from "../../../../../src/containers/app/content/home/eventCard";

describe('NavBar Component', () => {
  let component;

  beforeEach(() => {
    const props = {
      event: {
        id: 0,
        title: 'bla',
        img: 'https://www.pro-sky.com/images/en/news/news_header_webseite_-49-_342_big.jpg',
        description: 'blabla',
      },
      isLoading: false,
      onEventClick: jest.fn(),
    }
    component = renderer.create(
      <EventCard {...props}></EventCard>
    );
  });

  it('should render', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});