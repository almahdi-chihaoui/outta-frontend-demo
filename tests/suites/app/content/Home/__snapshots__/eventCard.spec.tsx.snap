// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Component should render 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <button
    className="MuiButtonBase-root MuiCardActionArea-root"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <div
      className="MuiCardMedia-root makeStyles-media-2"
      style={
        Object {
          "backgroundImage": "url(\\"https://www.pro-sky.com/images/en/news/news_header_webseite_-49-_342_big.jpg\\")",
        }
      }
      title=""
    />
    <div
      className="MuiCardContent-root"
    >
      <h2
        className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
      >
        bla
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
      >
        blabla
      </p>
    </div>
    <span
      className="MuiCardActionArea-focusHighlight"
    />
  </button>
  <div
    className="MuiCardActions-root MuiCardActions-spacing"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Share
      </span>
    </button>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Learn More
      </span>
    </button>
  </div>
</div>
`;
